app/javascript/topNavigation/utilities.js

```diff
@@ -52,18 +52,17 @@ export function initializeMemberMenu(memberTopMenu, menuNavButton) {
   if (navigator.userAgent === 'DEV-Native-ios') {
     document.body.classList.add('dev-ios-native-body');
   }
-
+  const { classList } = memberTopMenu;
   menuNavButton.addEventListener('click', (_event) => {
-    if (memberTopMenu.classList.contains('showing')) {
+    if (classList.contains('showing') && classList.contains('clicked')) {
```

Comment:
This is the core logic change.

Reply:
@djuber this is actually core logic for keeping the menu open when user clicked if the menu is already opened by hovered